GANDLF/utils.py

```diff
@@ -140,7 +140,7 @@ def reverse_one_hot(predmask_array, class_list):
     Returns:
         torch.Tensor: The final mask torch.Tensor.
     """
-    idx_argmax = np.argmax(predmask_array, axis=0)
+    idx_argmax = np.argmax(predmask_array.cpu().detach().numpy(), axis=0)
```

Comment:
Do we have to detach from graph?

Reply:
No, but it is the only thing that worked.

Reply:
Done.